src/hooks/useHashParams.ts

```diff
@@ -0,0 +1,14 @@
+export default function () {
+  const url = window.location.href
+  const regExStore = /store=(\w+|\w+\d+)/
+  const regExId = /id=(\d+)/
+
+  const matchStore = url.match(regExStore)
+  const matchId = url.match(regExId)
+
+  const object: { hashStore?: string; hashId?: string } = {}
```

Comment:
I think we shouldn't handle the case, when there's no store, or no id, only:
```TS
const object: { hashStore: string; hashId: string } | undefined
```
Empty object is nullish too, so you may use it

Reply:
Added `undefined`